src/main/java/Chris.java

```diff
@@ -0,0 +1,68 @@
+import java.util.Scanner;
+
+public class Chris {
+    protected static int taskCount = 0;
+    protected static Task[] taskList = new Task[100];
+    public static void main(String[] args) {
+        printLine();
+        System.out.println("Hello, I am Chris");
+        menu();
+        printLine();
+    }
+
+    public static void printLine() {
+        System.out.println("------------------------------------------------");
+    }
+
+    public static void printList() {
+        if (taskCount == 0) {
+            System.out.println("Sorry, there are no tasks currently.");
+            printLine();
+        } else {
+            System.out.println("Here are you current tasks!");
+            for (int i = 0; i < taskCount; i++) {
+                System.out.print(i + 1);
+                System.out.print(". ");
+                System.out.println(taskList[i]);
+            }
+        }
+    }
+
+    public static void menu() {
+        boolean done = false;
+        Scanner in = new Scanner(System.in);
+        while (!done) {
+            System.out.println("What can I do for you?");
+            printLine();
+            String line = in.nextLine();
+            if (line.startsWith("todo")) {
```

Comment:
Perhaps you might want to consider other ways to check for commands as "todoa" would likely trigger the set of code for "todo" keyword.